superset-frontend/src/components/ListView/TableCollection.tsx

```diff
@@ -29,6 +30,18 @@ interface Props {
   rows: TableInstance['rows'];
   loading: boolean;
 }
+
+const TableCell = styled.td`
```

Comment:
I see why you're adding `styled` here to apply props to the width, but there IS a possible performance implication here, if there wind up being a bajillion table cells, and Emotion starts bogging down. It may be more performant to use the prop to add a _class_ on the `td`, and then use emotion hire up in the DOM tree. I.e. 
```
const Table = styled.table`
  td{
    &.xs { width: 25px; }
    &.sm { width: 50px; }
    &.md { width: 75px; }
    &.lg { width: 100px; }
    &.xl { width: 150px; }
    &.xxl { width: 200px; }
  }
`
```